core/scheduler/service/job_sla_predictor_service.go

```diff
@@ -0,0 +1,294 @@
+package service
+
+import (
+	"context"
+	"time"
+
+	"github.com/goto/optimus/core/scheduler"
+	"github.com/goto/optimus/core/tenant"
+	"github.com/goto/salt/log"
+)
+
+type DurationEstimator interface {
+	GetP95DurationByJobNames(ctx context.Context, jobNames []scheduler.JobName, lastNRuns int, bufferPercentage int) (map[scheduler.JobName]*time.Duration, error)
```

Comment:
LastNRuns & BufferPercentage can be part of the in struct initilization, so that the interface can be clean otherwise we can't use the same interface.

Reply:
putting the `lastNruns` and `bufferpercentage` as a parameter here so that it's possible if we want to configure it at job level. is it ok to assume we'll never make this configuration at job level?